daemon/commands.c

```diff
@@ -411,7 +412,7 @@ static inline void add_string_to_command_reply(char *reply_string, unsigned *rep
 static void send_command_reply(struct command_context *cmd_ctx, cmd_status_t status, char *message)
 {
     int ret;
-    char reply_string[MAX_COMMAND_LENGTH] = {'\0', };
+    char *reply_string = mallocz(message ? (strlen(message) + MAX_EXIT_STATUS_LENGTH + 2) : MAX_COMMAND_LENGTH);
```

Comment:
`if(!message)` why use MAX_COMMAND_LENGTH wouldn't then be enough to allocate `MAX_EXIT_STATUS_LENGTH + 2` ?

Reply:
will allocate MAX_COMMAND_LENGTH to keep it simple

Reply:
Assigned